src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -74,8 +74,8 @@
     <PackageReference Include="Microsoft.Azure.AppService.Middleware.Functions" Version="1.4.17" />
     <PackageReference Include="Microsoft.Azure.Cosmos.Table" Version="1.0.8" />
     <PackageReference Include="Microsoft.Azure.Storage.File" Version="11.1.7" />
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.34-11937" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Host.Storage" Version="5.0.0-beta.1" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.35-11939" />
```

Comment:
Not sure we should be at minor version 35. In [this](https://github.com/Azure/azure-webjobs-sdk/commit/71c2262d8bf64fb845429c557732f9be0e7b24a8) PR I recently bumped us to 34 and we haven't officially released that yet, so we should still be in the 34 minor version train.

Reply:
Reverting to 34!